src/Uno.UI/UI/Xaml/Automation/Peers/AutomationPeer.cs

```diff
@@ -106,6 +106,8 @@ public partial class AutomationPeer : DependencyObject
 
 		public AutomationPeer GetLabeledBy() => GetLabeledByCore();
 
+		protected internal IRawElementProviderSimple ProviderFromPeer(AutomationPeer peer) => new IRawElementProviderSimple(peer);
```

Comment:
@MartinZikmund The method is redefined in the same file. Could you fix that and see if this will get green?